test/integration/test_itde_manager_in_docker_container.py

```diff
@@ -231,12 +231,45 @@ def run_test():
     return source_code
 
 
+@pytest.fixture
+def itde_external_test():
+    """
+    This fixture returns the test source code for testing the use of an externally created ITDE. The test
+    checks that it survives the calls bring_itde_up/take_itde_down with an indication of using an external ITDE.
+    The source code needs to appended to the wheel file inside the Docker container called TEST_CONTAINER.
+    """
+
+    def run_test():
+        from pathlib import Path
+
+        from exasol_integration_test_docker_environment.lib import api
+        from exasol.nb_connector.connections import open_pyexasol_connection
+        from exasol.nb_connector.itde_manager import bring_itde_up, take_itde_down
+        from exasol.nb_connector.secret_store import Secrets
+
+        secrets = Secrets(db_file=Path("secrets.sqlcipher"), master_password="test")
+        env_info, cleanup_func = api.spawn_test_environment(environment_name="TestDemoDb")
+        try:
+            bring_itde_up(secrets, env_info)
```

Comment:
Maybe also check that it doesn't create a container DemoDB and instead actually points to TestDemoDB

Reply:
See the answer to the next comment